IBPSA/Fluid/HeatPumps/ModularReversible/Data/TableData2D/GenericHeatPump.mo

```diff
@@ -0,0 +1,43 @@
+within IBPSA.Fluid.HeatPumps.ModularReversible.Data.TableData2D;
+record GenericHeatPump "Partial heat pump data"
+  extends IBPSA.Fluid.HeatPumps.ModularReversible.Data.TableData2D.Generic;
+  parameter Real tabQCon_flow[:,:]
+    "Heating output table, T in degC, Q_flow in W";
```

Comment:
Let's not use degC. Use K, otherwise it will be a disaster as users for sure mix up units.